src/Tools/RulesetToEditorconfigConverter/Source/Program.cs

```diff
@@ -1,42 +1,37 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#pragma warning disable IDE0051 // Remove unused private members
```

Comment:
Isn't this bug fixed yet?

Reply:
It was still getting reported in 16.7 Preview 3. I'm assuming that dotnet/roslyn#44848 will allow this to be cleaned up when the IDE is updated.

Reply:
Looks like https://github.com/dotnet/roslyn/issues/45496 might not be fixed yet.